test-app/src/widgets/ListBox.js

```diff
@@ -26,6 +26,14 @@ export default class ListBox extends createjs.Container {
     this._updateSelectedOption(selectedOption);
   }
 
+  setTabbable(tabbable) {
```

Comment:
(Same as combobox) Added so the tabindex can be flipped when the listbox shouldn't be in the tab order; in our case this is when the dialog is open in the Form example and the listbox is behind it